test/support/ssl/insecure-cert.pem

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
+MIIF9DCCA9ygAwIBAgIJAJhLP5JrvONkMA0GCSqGSIb3DQEBBQUAMFkxCzAJBgNV
```

Comment:
do we need a test key and cert for testing this or will chrome throw that certificate error without it?